packages/components/src/private-apis.ts

```diff
@@ -39,6 +40,7 @@ lock( privateApis, {
 	CompositeItemV2,
 	CompositeRowV2,
 	useCompositeStoreV2,
+	CustomSelectControlV2LegacyWrapper,
```

Comment:
Naming nit: is this really a wrapper though? Since the legacy component isn't really a wrapper as part of a composable component group, but rather a monolithic component, I'd prefer naming this `CustomSelectControlV2Legacy` or `CustomSelectControlV2LegacyAdapter` since `Wrapper` implies a wrapper component in the same way the `Tabs` parent component wraps the rest of the subcomponents. 

No strong feelings about it, but I feel like others might potentially be confused by that naming.

Reply:
Good point, pushed an update